packages/paste-website/src/components/icons-list/IconsList.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import {useUID, useUIDSeed} from 'react-uid';
+import {styled} from '@twilio-paste/styling-library';
+import {Box} from '@twilio-paste/box';
+import {Grid, Column} from '@twilio-paste/grid';
+import {FormLabel, FormInput} from '@twilio-paste/form';
+import {Heading} from '@twilio-paste/heading';
+import {IconTile} from './IconTile';
+import {IconObject, IconComponent, IconsListProps, GroupedList} from './types';
+import {IconCard} from './IconCard';
+import {getStickyColumnComputedOffset} from '../../utils/stickyUtils';
+import {SiteBodyContext} from '../site-wrapper/SiteBody';
+
+const iconsJson = require('@twilio-paste/icons/json/icons.json');
+
+const IconComponents: IconComponent = iconsJson.reduce((icons: IconComponent, {name}: IconObject) => {
+  return {
+    ...icons,
+    // eslint-disable-next-line import/no-dynamic-require,global-require
+    [`${name}`]: require(`@twilio-paste/icons/esm/${name}`)[name],
+  };
+}, {});
+
+const getGroupedList = (icons: IconsListProps['icons']): GroupedList =>
+  icons.reduce(
+    (prev: GroupedList, current): GroupedList => {
+      const Icon = {...current, component: IconComponents[current.name]};
+      if (current.name.startsWith('Product')) {
+        return {
+          products: [...prev.products, Icon],
+          ui: [...prev.ui],
+        };
+      }
+      return {
+        products: [...prev.products],
+        ui: [...prev.ui, Icon],
+      };
+    },
+    {products: [], ui: []}
+  );
+
+const getFirstIcon = (iconsList: GroupedList): IconObject | null => {
+  if (iconsList.ui.length !== 0) {
+    return iconsList.ui[0];
+  }
+  if (iconsList.products.length !== 0) {
+    return iconsList.products[0];
+  }
+  return null;
+};
+
+const IconTileWrapper = styled(Box)({
+  width: '33.33%',
```

Comment:
🤔

Reply:
Yeah, our Grid doesn't wrap in the way I was expecting because we calculate the number of columns base on number of children. And our box widths only takes tokens.

Aside, increasingly convinced that height and width should be treated like top, bottom, right and, left.

Reply:
I'm into that tbh